src/components/BackstageComparison/FeatureComparisonTable.js

```diff
@@ -0,0 +1,413 @@
+        name: 'Fallback and retry',
+        competitor: { Roadie: true, Backstage: true },
+      },
+    ],
+  },
+  {
+    name: 'TechDocs',
+    features: [
+      {
+        name: 'Docs creation engine',
+        competitor: { Roadie: 'TechDocs', Backstage: 'TechDocs' },
+      },
+      {
+        name: 'Extensibility via plugins',
+        competitor: { Roadie: true, Backstage: true },
+      },
+    ],
+  },
+  {
+    name: 'Other plugins and extensions',
+    features: [
+      {
+        name: 'Available Plugins',
+        competitor: { Roadie: '70+', Backstage: 'Requires build' },
+      },
+      {
+        name: 'Access to on-prem infrastructure and resources',
+        competitor: { Roadie: true, Backstage: true },
+      },
+      {
+        name: 'Theme customisation',
+        competitor: { Roadie: true, Backstage: 'Requires build' },
+      },
+      {
+        name: 'Layout customisation',
+        competitor: { Roadie: true, Backstage: 'Requires build' },
+      },
+    ],
+  },
+  {
+    name: 'Search',
+    features: [
+      {
+        name: 'Engine',
+        competitor: { Roadie: 'Backstage search', Backstage: 'Backstage search' },
+      },
+      {
+        name: 'Hosted search collators',
+        competitor: { Roadie: 'Atlassian Confluence', Backstage: false },
+      },
+    ],
+  },
+  {
+    name: 'Users and Groups',
+    features: [
+      {
+        name: 'SSO',
+        competitor: { Roadie: true, Backstage: 'Via OSS providers' },
+      },
+      {
+        name: 'Entities created in the Catalog for each user and group',
+        competitor: { Roadie: true, Backstage: 'Via OSS providers' },
+      },
+    ],
+  },
+  {
+    name: 'Role-based Access Control',
+    features: [
+      {
+        name: 'Admin and non-Admin roles',
+        competitor: { Roadie: true, Backstage: true },
+      },
+      {
+        name: 'Role assignment in the UI',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Role assignment by identity provider token',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Custom roles',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Custom permissions policies',
+        competitor: { Roadie: true, Backstage: false },
+      },
+    ],
+  },
+  {
+    name: 'Non-feature extras',
+    features: [
+      {
+        name: 'Hosting',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Upgrades to the latest version of Backstage',
+        competitor: { Roadie: true, Backstage: 'Requires build' },
+      },
+      {
+        name: 'Analytics',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Support',
+        competitor: {
+          Roadie: 'Email, Slack/Teams, Discord, and 24/7 On-call',
+          Backstage: 'via the OSS community',
+        },
+      },
+      {
+        name: 'Cost',
```

Comment:
What does this one mean?

Reply:
I was going to link out to a cost comparison, but feels non-trivial for the first pass. i've removed